scripts/update-xliff.py

```diff
@@ -50,15 +50,23 @@ def main():
 
     parser = argparse.ArgumentParser()
     parser.add_argument('base_folder', help='Path to folder including subfolders for all locales')
+    parser.add_argument('locales', default='*', nargs='*', help='Locales to process (default: %(default)s)')
```

Comment:
It's basically used if no argument is provided. Anyway, I've tested this PR on Debian and these commands are equivalent
```
update-xliff.py ~/path/to/l10n-repo
update-xliff.py ~/path/to/l10n-repo *
```